crates/interpreter/src/instructions.rs

```diff
@@ -159,7 +220,7 @@ const fn instruction_table_impl<WIRE: InterpreterTypes, H: Host>() -> [Instructi
     table[JUMPDEST as usize] = Instruction::new(control::jumpdest, 1);
     table[TLOAD as usize] = Instruction::new(host::tload, 100);
     table[TSTORE as usize] = Instruction::new(host::tstore, 100);
-    table[MCOPY as usize] = Instruction::new(memory::mcopy, 0); // static 2, mostly dynamic
+    table[MCOPY as usize] = Instruction::new(memory::mcopy, 3); // static 2, mostly dynamic
```

Comment:
MCOPY has a static VERY_LOG gas 

https://github.com/bluealloy/revm/blob/d0bb48ef6ebf6db67df0f9409408b2afc73241f2/crates/interpreter/src/instructions/memory.rs#L68

VERYLOW is 3gas.
https://github.com/bluealloy/revm/blob/d0bb48ef6ebf6db67df0f9409408b2afc73241f2/crates/interpreter/src/gas/calc.rs#L110-L111